src/includes/Helpers/ConstantsHelper.php

```diff
@@ -73,6 +73,8 @@ class ConstantsHelper {
 
 	const ERROR = 'error';
 
+	const COLLECT_URL = 'alma_collect_data_url';
```

Comment:
I think you can move this const in service

Reply:
Done